src/pages/guides/using/aem/index.md

```diff
@@ -1,24 +1,25 @@
 ---
-title: AEM Events
+title: Setting up AEM Events with Adobe IO Events
 ---
 
-# AEM Events
+# Setting up AEM Events with Adobe IO Events
 
-**Adobe I/O Events integration package for AEM is now open source:
```

Comment:
I prefer the old wording here

Reply:
You mean to get rid of `These instructions describe how to set up Adobe Experience Manager (AEM) to integrate with Adobe IO Events.`?

Reply:
Reverted back to old wording.